app/components/Intercept_Components/InterceptTextBox.tsx

```diff
@@ -18,13 +25,27 @@ export const InterceptTextBox = props => {
         </a>
       </div>
       <div className="response">
-        <label htmlFor="">Response Text</label>
+        <label className="responseTextlabel">Response Text</label>
+        <span
+          title="Fetch Response Text"
```

Comment:
why not use a `<a>` or a `button` tag for this? Also, title here doesn't seem to suggest what the action does

Reply:
Yeah, wanted that to replace that with an icon before the release. Just put up a placeholder `span` tag for now